terraform/environments/ccms-ebs/iam.tf

```diff
@@ -0,0 +1,91 @@
+## SSM Specific
+resource "aws_iam_policy" "oracle_ec2_ssm_policy" {
```

Comment:
Hi,
Can we rename the respurce name to make it generic so that these can be used with other resources as well
oracle_ec2_ssm_policy >> ec2_ssm_policy